lms/djangoapps/branding/api.py

```diff
@@ -443,15 +444,32 @@ def _footer_mobile_links(is_secure):
 
 
 def _footer_logo_img(is_secure):
-    """Return the logo used for footer about link
+    """
+    Return the logo used for footer about link
 
-    Args:
+    Arguments:
         is_secure (bool): Whether the request is using TLS.
 
     Returns:
-        Absolute url to logo
+        URL of the brand logo
     """
-    logo_name = configuration_helpers.get_value('FOOTER_ORGANIZATION_IMAGE', settings.FOOTER_ORGANIZATION_IMAGE)
+    default_local_path = 'images/logo.png'
+    brand_footer_logo_url = settings.LOGO_TRADEMARK_URL
+    footer_url_from_site_config = configuration_helpers.get_value(
```

Comment:
nit: site_config_footer_url seems easy to read.

Reply:
+1 on that.